src/commands/backup.rs

```diff
@@ -72,6 +72,27 @@ pub struct BackupCmd {
     #[merge(strategy=conflate::bool::overwrite_false)]
     json: bool,
 
+    /// Push metrics to a Prometheus Pushgateway
+    ///
+    /// If the Pushgateway requires authentication, then the login/password can be passed in
+    /// the `PUSHGATEWAY_USER` and `PUSHGATEWAY_PASS` environment variables.
+    #[cfg(feature = "prometheus")]
+    #[clap(long, value_name = "PUSHGATEWAY_URL", value_hint = ValueHint::Url)]
+    #[merge(strategy=conflate::option::overwrite_none)]
+    prometheus: Option<String>,
+
+    /// Job name for the Pushgateway push
+    #[cfg(feature = "prometheus")]
+    #[clap(long, value_name = "JOB_NAME", default_value = "rustic")]
+    #[merge(skip)]
+    prometheus_job: String,
```

Comment:
with `merge(skip)`, this won't be configurable via the config profiles. Better also use a `Option<String>` with `overwrite(none)` merge strategy and define a default value for `None`.

Reply:
Done :)